math_coding/flows/math_standard_flow/flow.dag.yaml

```diff
@@ -1,5 +1,6 @@
 $schema: https://azuremlschemas.azureedge.net/promptflow/latest/Flow.schema.json
 environment:
+  image: fauxdatamesh.azurecr.us/promptflow/promptflow-runtime-gov
```

Comment:
flagging that this is hardcoded... opportunity to make it an ENV or injectable? maybe not needed for now?

Reply:
Thanks for reminding me, this was my temp solution yesterday but isn't actually needed anymore